src/core/runtime/libp2p-browser.js

```diff
@@ -51,8 +53,12 @@ class Node extends libp2p {
             enabled: true
           }
         },
+        dht: {
+          kBucketSize: 20,
+          enabledDiscovery: true
+        },
         EXPERIMENTAL: {
-          dht: false,
+          dht: true,
```

Comment:
No need anymore, right?

Reply:
In the `libp2p` side, it is still marked as experimental. I am totally up to make it enabled by default and running if `libp2p` receives the DHT in the modules object.

Should we release a new version of `libp2p` removing the experimental flag now? If you all agree, I will make the PR later today.

cc @jacobheun

Reply:
> Should we release a new version of libp2p removing the experimental flag now? If you all agree, I will make the PR later today.

I agree 👍 💯

Reply:
I also agree, set it free!